src/data/list/basic.lean

```diff
@@ -601,7 +601,7 @@ theorem init_append_last : ∀ {l : list α} (h : l ≠ []), init l ++ [last l h
 begin
   rw [init, cons_append, last_cons (cons_ne_nil _ _) (cons_ne_nil _ _)],
   congr,
-  exact init_append_last (cons_ne_nil b l),
+  exact init_append_last (cons_ne_nil b l)
```

Comment:
Just FYI, the trailing comma is allowed, depending on your taste: https://leanprover-community.github.io/contribute/style.html#tactic-mode